web/src/main/angular/src/app/core/components/transaction-id-search/transaction-id-search-container.component.ts

```diff
@@ -0,0 +1,53 @@
+import {
+    Component,
+    ChangeDetectionStrategy,
+    Output,
+    EventEmitter
+} from '@angular/core';
+
+import {
+    UrlRouteManagerService
+} from 'app/shared/services';
+import { UrlPath } from 'app/shared/models';
+
+@Component({
+    selector: 'pp-transaction-id-search-container',
+    templateUrl: './transaction-id-search-container.component.html',
+    styleUrls: ['./transaction-id-search-container.component.css'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class TransactionIdSearchContainerComponent {
+    @Output() outSearchId = new EventEmitter<{txId: string}>();
+
+    constructor(
+        private urlRouteManagerService: UrlRouteManagerService,
+    ) {}
+
+    onSearchId(txId: string): void {
+        if (txId === '') {
+            return;
+        }
+
+        const spanId = '-1';
+
+        const collectorAcceptTime = '0';
+
+        const txInfo = txId.split('^');
+
+        if (txInfo.length !== 3) {
+            return;
+        }
```

Comment:
If variable declarations are continuous, no line-break. Sorry if I misled you.
```suggestion
        const spanId = '-1';
        const collectorAcceptTime = '0';
        const txInfo = txId.split('^');

        if (txInfo.length !== 3) {
            return;
        }
```